local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -228,6 +233,16 @@ void LocalPlannerNode::updatePlannerInfo() {
     local_planner_.setGoal(goal_msg_.pose.position);
     new_goal_ = false;
   }
+
+  // update ground distance
+  if (ros::Time::now() - ground_distance_msg_.header.stamp <
+      ros::Duration(0.5)) {
+    local_planner_.ground_distance_ = ground_distance_msg_.bottom_clearance;
+  } else {
+    local_planner_.ground_distance_ = 2.0;  // in case where no range data is
```

Comment:
So we consider 2.0 meters "close"? I'd call 2 cm close. Am I correctly assuming measurement in meters? I'd prefer if the variable name reflected the units...